src/main/java/hudson/plugins/s3/S3BucketPublisher.java

```diff
@@ -119,7 +127,9 @@ public static S3Profile getProfile(String profileName) {
     }
 
     private void log(final PrintStream logger, final String message) {
-        logger.println(StringUtils.defaultString(getDescriptor().getDisplayName()) + ' ' + message);
+        if(! suppressLogging) {
```

Comment:
I've addressed this in my latest commit. Let me know if you like the implementation.